Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -483,6 +483,8 @@ inline LengthPoint BuilderConverter::convertPositionOrAuto(BuilderState& builder
 {
     if (value.isPair())
         return convertPosition(builderState, value);
+    if (value.valueID() == CSSValueNormal)
+        return LengthPoint({ LengthType::Normal }, { LengthType::Normal });
```

Comment:
Should be able to use braces without types like this `{ { LengthType::Normal }, { LengthType::Normal } }` or even find a way to share code with `RenderStyle::initialOffsetPosition` to avoid repeating a long expression.

Reply:
Fixed.